File: app/src/main/java/domain/ExternalIds.java
package domain;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.io.Serializable;

public class ExternalIds implements Serializable {

    @JsonProperty("imdb_id")
    private String imdbId;

    @JsonProperty("freebase_id")
    private String freeBaseId;

    @JsonProperty("freebase_mid")
    private String freebaseMid;

    @JsonProperty("tvdb_id")
    private String tvdbId;

    @JsonProperty("tvrage_id")
    private String tvrageId;

    public void setImdbId(String imdbId) {
        this.imdbId = imdbId;
    }

    public void setFreeBaseId(String freeBaseId) {
        this.freeBaseId = freeBaseId;
    }

    public void setFreebaseMid(String freebaseMid) {
        this.freebaseMid = freebaseMid;
    }

    public void setTvdbId(String tvdbId) {
        this.tvdbId = tvdbId;
    }

    public void setTvrageId(String tvrageId) {
        this.tvrageId = tvrageId;
    }

    public String getImdbId() {
        return imdbId;
    }


    public String getFreeBaseId() {
        return freeBaseId;
    }


    public String getFreebaseMid() {
        return freebaseMid;
    }


    public String getTvdbId() {
        return tvdbId;
    }


    public String getTvrageId() {
        return tvrageId;
    }
}

File: app/src/main/java/domain/tvshow/ExternalIds.kt
package domain.tvshow

import com.google.gson.annotations.SerializedName
import java.io.Serializable
import javax.annotation.Generated

@Generated("com.robohorse.robopojogenerator")
data class ExternalIds(

	@field:SerializedName("imdb_id")
	var imdbId: String? = null,

	@field:SerializedName("freebase_mid")
	var freebaseMid: String? = null,

	@field:SerializedName("tvdb_id")
	var tvdbId: String? = null,

	@field:SerializedName("freebase_id")
	var freebaseId: String? = null,

	@field:SerializedName("tvrage_id")
	var tvrageId: String? = null

): Serializable